lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -715,39 +689,39 @@ def poly_collection_2d_to_3d(col, zs=0, zdir='z'):
     col.set_verts_and_codes(segments_3d, codes)
     col.set_3d_properties()
 
-def juggle_axes(xs, ys, zs, zdir):
+def juggle_axes(xyz, zdir):
```

Comment:
Since I changed all the calls to this function, I thought that it would be OK to force people using a more optimized one. Will revert.